desktop/extensions-sdk/architecture/metadata.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Extension metadata
+description: Docker extension metadata
+keywords: Docker, extensions, sdk, metadata
+redirect_from:
+- /desktop/extensions-sdk/extensions/metadata
+---
+
+## The metadata.json file
+
+The `metadata.json` file is the entry point for your extension. It contains the metadata for your extension, such as the
+name, version, and description. It also contains the information needed to build and run your extension. The image for
+a Docker extension must include a `metadata.json` file at the root of its filesystem.
+
+The format of the `metadata.json` file must be:
+
+```json
+{
+    "icon": "extension-icon.svg",
+    "ui": ...
+    "vm": ...
+    "host": ...
+}
+```
+
+The `ui`, `vm`, and `host` sections are optional and depend on what a given extension provides. They describe the extension content to be installed.
+
+### <a name="ui"></a>UI section
```

Comment:
@benja-M-1 Looks like this is newly added. Do we need an anchor link here? Wondering if we can use a heading instead?